api/tests/test_cloud_record_summary.py

```diff
@@ -127,3 +250,75 @@ def test_filter_cursor(self):
     def tearDown(self):
         """Delete any messages under QPATH and re-enable logging.INFO."""
         logging.disable(logging.NOTSET)
+
+    def _populate_database(self, database):
```

Comment:
Would be nicer if these helper methods could be used within a context hander, but probably not worth the effort for a single test. However it would be better to at least have the final `_clear_database` and `database.close` calls above be at the end of a `try...finally` so that it always executes, even if the rest of the code fails.